pom.xml

```diff
@@ -34,6 +34,7 @@
         <jetty.version>9.4.6.v20170531</jetty.version>
         <jersey.version>2.26-b06</jersey.version>
         <log4j.version>2.8.2</log4j.version>
+        <jackson-2-version>2.8.1</jackson-2-version>
```

Comment:
This should probably just be `jackson.version`; there's no situation where we should have both Jackson 2 and 1.